pom.xml

```diff
@@ -35,11 +35,11 @@
 
         <!-- On this block, library versions must be kept in sync with the ones in the quarkus-bom: -->
         <!-- Start-->
-        <quarkus.version>3.13.0.CR1</quarkus.version>
+        <quarkus.version>3.16.1</quarkus.version>
```

Comment:
We should probably update to 3.15.x first. It is LTS and corporate users will likely choose it

Reply:
So, I have first bump Quarkus to the last LTS and removed the OpenTelemetry bump.

There is a GCP issue. I'm not supposed to work on GCP today. ;) @brunobat Could you please have a look?